### ISSUE TITLE
Not generating proper results

### ISSUE THREAD

**Issue by shetsachin** [Microsoft/StorScore#42]
Hi,
 Initially I started by running the 4k_RR.rcp on my SSD and I got the results in the excel sheet with all the details but later when I ran Storscore command with 4K_RW.rcp or any other recipes, then only the first sheet is filled with the data. Final and score details looks empty even after many runs.

Commands used to run:
 StorScore.cmd -recipe=recipes\4k_RW.rcp --target=f:
 parse_results.cmd results* results.xlsx

Can anyone guide me/tell me why I am not getting the full result sheet with all the data even after parsing?

**Comment by marksantaniello** [comment]
Yeah this is one of the more confusing things about StorScore.  The scoring system is based entirely on comparing a set of devices against each other, sort of like grading on a curve.  With only a single device, the scores are meaningless.  At Microsoft we tested many comparable devices together -- often a dozen or more -- with the goal of selecting the best two or three.

**Comment by shetsachin** [comment]
But still even if we use a single device we should get the data filled with that particular device scores. As I don have multiple devices, currently running StorScore on single device. And facing above mentioned issue.
could you tell me the command to run multiple devices in a single command?

**Comment by marksantaniello** [comment]
You just pass a set of results directories to the parser, so you can say:
`parse_results.cmd results\drive1 results\drive2 output`

**Comment by shetsachin** [comment]
Thank you..
But still my issue remains same. Is there any solution you can suggest me to get the full result?

**Comment by marksantaniello** [comment]
I guess I'm still not 100% sure I understand the problem.  

There's no way to generate scores for a single device, using our method.  We use Z-scores, which are just the number of standard deviations away from the mean.  For a single device the Z-score is always zero.  You can get IOPS, and MB/sec, and latencies, etc. -- but you can't get scores, because those are defined by the comparison with other devices.  

Maybe you can attach the XLSX and give a very clear description of what you expected to see?
